packages/client/src/__tests__/integration/happy/blog-env-mssql/test.ts

```diff
@@ -270,5 +270,17 @@ describe('blog-env-mssql', () => {
         await prisma.$executeRaw`SELECT * FROM [dbo].[User] WHERE name = ${'F'}`
       expect(users).not.toBe(0)
     })
+
```

Comment:
```suggestion
})

describe('$*Raw errors', () => {
```

Reply:
i don't like it 😄 it's not great for doing a search

Reply:
You can spell it both out of course